frontend/add-leads/add-edit-materialEntry.css

```
@@ -0,0 +1,20 @@
+.container {
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.deleteButton {
+  color: red;
```

Let's use some colors from the styleguide / brand.

```suggestion
  color: var(--medium-red);
```

See https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/b331fd9acaf3797ba5239096b27356a846cc8cd7/frontend/styleguide.component.tsx#L34